elementary/monitor/dbt_project/models/tests_recommendation/test_recommendations.sql

```diff
@@ -72,14 +82,23 @@ with
             exposure_count,
             table_type,
             case
-                when timestamp_column is not null
+                when short_name in ('volume_anomalies', 'freshness_anomalies') and timestamp_column is not null
                 then cast('{"timestamp_column": "' || timestamp_column || '"}' as jsonb)
-                else null
-            end as test_args
+            end as test_args,
+            case
+                when short_name = 'schema_changes_from_baseline'
+                then cast(json_build_object('columns', table_columns.columns) as jsonb)
+            end as table_args
         from pending_recommended_tests
         join tables_criticality using (id)
         left join timestamp_columns using (database_name, schema_name, table_name)
-        where requires_timestamp_column = false or timestamp_column is not null
+        left join table_columns using (database_name, schema_name, table_name)
+        where
+        short_name = 'volume_anomalies'
+        or
+        short_name = 'freshness_anomalies' and timestamp_column is not null
```

Comment:
add  `()` around

Reply:
Why? `and` evaluates before `or`.

Reply:
im not sure about it, i believe its just in sequential order, but thats exactly the point - it just more explicit with `()` (i like explicit)